src/bci_build/package/templates/badges.j2

```diff
@@ -1,6 +1,12 @@
 {% if image.eula == 'sle-bci' -%}
 ![Redistributable](https://img.shields.io/badge/Redistributable-Yes-green)
 {%- endif %}
-{% if not image.is_opensuse and image.support_level | string != 'l3' -%}
+{%- if not image.is_opensuse and image.support_level | string != 'l3' -%}
 ![Support Level](https://img.shields.io/badge/Support_Level-{{ image.support_level }}-blue)
+{%- endif %}
+{%- if not image.is_opensuse and image.build_tags[0].find('/ltss/') != -1 -%}
+![Access Protected](https://img.shields.io/badge/Requires_login_for_access-orange)
+{%- endif %}
+{%- if not image.is_opensuse and image.build_tags[0].find('/ltss/') != -1 -%}
+![Long Term Service Pack Support](https://img.shields.io/badge/LTSS-Yes-orange)
```

Comment:
Duplicate if?
```suggestion
{%- if not image.is_opensuse and image.build_tags[0].find('/ltss/') != -1 -%}
![Access Protected](https://img.shields.io/badge/Requires_login_for_access-orange)
![Long Term Service Pack Support](https://img.shields.io/badge/LTSS-Yes-orange)
```

Also, this looks to me like it could be worth adding a property `is_ltss` to either the image or the `OsVersion`. WDYT?

Reply:
I intentionally duplicated the if because eventually we'll have to have a different if condition for "access protected" and "ltss" since we'll introduce more access protected images going forward.